resources/deprecations.properties

```diff
@@ -125,7 +135,12 @@ origo-issue-notifier = https://wiki.jenkins.io/x/qgabAg
 paranoia = https://git.io/JfaQS
 # https://plugins.jenkins.io/perfectomobile/
 perfectomobile = https://developers.perfectomobile.com/display/PD/Legacy+%7C+Jenkins+plugin
+perforce = https://jenkins.io/security/advisory/2018-03-26/
```

Comment:
Re https://github.com/jenkins-infra/jenkins.io/pull/4359#issuecomment-841615950,

```suggestion
perforce = https://www.jenkins.io/security/advisory/2018-03-26/#SECURITY-373
```

Looks like a reasonable(ish) reference in the description there.